tools/toolchains/arm.py

```diff
@@ -418,20 +423,27 @@ def __init__(self, target, *args, **kwargs):
             self.flags['common'].append("-march=armv8-m.base")
         elif target.core.startswith("Cortex-M33F"):
             self.flags['common'].append("-mfpu=fpv5-sp-d16")
-            self.flags['common'].append("-mfloat-abi=softfp")
+            self.flags['common'].append("-mfloat-abi=hard")
```

Comment:
I am not sure of `hard` floating point. @TacoGrandeTX @theotherjimmy - Do you have any suggestions for floating option in ARMv8M? I see we have always selected `soft` floating point for v6 / v7 architecture.

Reply:
We have selected `softfp`